app/views/posts/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+    <h1>New Post</h1>
+    <%= form_with model: @new_post, url: user_posts_path(@current_user) do |form| %> 
+        <div class="form-group">
+            <%= form.label :title %>
+            <%= form.text_field :title, class: "form-control" %>
+        </div>
+        <div class="form-group">
+            <%= form.label :text %>
+            <%= form.text_area :text, size:"100x10", class: "form-control" %>
+        </div>
+        <% if flash.now[:alert] %>
+            <p class="alert"><%= flash[:alert] %></p>
+        <% end %>
+        <%= form.submit "Submit", class: "btn btn-primary" %>
+    <% end %>
```

Comment:
- [ ] I noticed that the functionality to create a new post for the `current_user` is not working correctly. When the submit button is pressed, the page reloads but the data is not persisted to the database.
As a result of this, the comment creation and likes functionality have not been tested. Please fix this issue 👍 

https://github.com/AlaaAlsalem/blogApp/assets/46839548/639386f0-d5d3-471b-adb6-c08adf59d7e3